### ISSUE TITLE
Incompatibility with Ansible cron jobs

### ISSUE THREAD

**Issue by jadami10** [doctormo/python-crontab#13]
When creating cron jobs with ansible, there's always a commented line before the job.
```
#Ansible: {job_name}
* * * * * {command}
```
But any time this format is read in by CronTab, the line is parsed as 
```
* * * * * {command} #Ansible: {job_name}
```
This causes CronTab to write it back out in this new format again which causes ansible to think the job is missing and write it out again. And if both ansible and something using CronTab are running regularly, you end up with a file filled with the same command.

**Comment by doctormo** [comment]
Interesting. Thanks for reporting this error. I will have to think about a solution.

**Comment by doctormo** [comment]
Fixed in 2.2.7

**Comment by jadami10** [comment]
Hey, is this code going to be added to this repo?
